### ISSUE TITLE
ockafka does not seem to build

### ISSUE THREAD

**Issue by jorisc90** [aristanetworks/goarista#20]
As I'm not the biggest Go hero around, I can't seem to resolve this issue while building ockafka. Any suggestions or is there a bug that needs solving?
```
joris@lab:~$ go install github.com/aristanetworks/goarista/cmd/ockafka
# github.com/aristanetworks/goarista/monitor
go/src/github.com/aristanetworks/goarista/monitor/map.go:24: undefined: sync.Map
# github.com/Shopify/sarama
go/src/github.com/Shopify/sarama/delete_records_request.go:96: undefined: sort.Slice
go/src/github.com/Shopify/sarama/delete_records_response.go:99: undefined: sort.Slice
```

**Comment by 7AC** [comment]
Is it possible you're using an old Go version (< 1.8)? Those types were released in Go 1.9 and 1.8 respectively:

https://golang.org/doc/go1.9#sync-map
https://golang.org/doc/go1.8#sort_slice

**Comment by jorisc90** [comment]
I got burned by using Debian (again)... Going to rebase my lab server on Ubuntu shortly I guess.
